modules/schedule/src/test/java/org/apache/ignite/internal/processors/schedule/GridScheduleSelfTest.java

```diff
@@ -120,7 +121,7 @@ public void testScheduleRunnable() throws Exception {
 
         SchedulerFuture<?> fut = null;
 
-        long freq = 60; // 1 minute frequency.
+        long freq = 2; // 2 seconds frequency.
```

Comment:
May be it will be better to stay old tests as-is to make sure new implementation works the same way?
And add new tests?

Reply:
Agree. It looks we don't need a new test here.